api/app/services/training_service.py

```
@@ -0,0 +1,179 @@
+        return self._job_to_response(job)
+
+    async def stop_job(self, job_id: str) -> TrainingJobData:
+        """Stop a running training job."""
+        job = await self.job_manager.stop_job(job_id)
+        return self._job_to_response(job)
+
+    def get_job_logs(
+        self,
+        job_id: str,
+        level: Optional[str] = None,
+        since: Optional[datetime] = None,
```

Ruff (0.14.8)

Unused method argument: `since`

(ARG002)